libmamba/src/core/virtual_packages.cpp

```diff
@@ -190,13 +190,13 @@ namespace mamba
                 }
             }
 
-            LOG_WARNING << "Could not find CUDA version by, in this order:\n";
-            LOG_WARNING << " - inspecting the `CONDA_OVERRIDE_CUDA` environment variable\n";
-            LOG_WARNING << " - parsing the : " << cuda_version_file << "\n";
-            LOG_WARNING << " - parsing the output of `nvidia-smi --query -u -x`\n";
-            LOG_WARNING << "\n";
-            LOG_WARNING << "We recommend setting the `CONDA_OVERRIDE_CUDA` environment variable\n";
-            LOG_WARNING << "to the desired CUDA version.";
+            LOG_DEBUG << "Could not find CUDA version by, in this order:\n";
+            LOG_DEBUG << " - inspecting the `CONDA_OVERRIDE_CUDA` environment variable\n";
+            LOG_DEBUG << " - parsing the : " << cuda_version_file << "\n";
+            LOG_DEBUG << " - parsing the output of `nvidia-smi --query -u -x`\n";
+            LOG_DEBUG << "\n";
+            LOG_DEBUG << "We recommend setting the `CONDA_OVERRIDE_CUDA` environment variable\n";
+            LOG_DEBUG << "to the desired CUDA version.";
```

Comment:
We can have these with LOG_TRACE if you think it could be useful, but previous logs could be enough and these could be removed.
But in any case, I would add back:
`LOG_DEBUG << "CUDA not found";`

Reply:
```suggestion
            LOG_DEBUG << "CUDA not found";
```